.github/workflows/build.yml

```
@@ -0,0 +1,23 @@
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        java: [17, 21]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: ${{ matrix.java }}
+      - uses: gradle/gradle-build-action@v2
+      - run: ./gradlew clean build
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)